src/delphi/llama2.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+from llama2c.model import ModelArgs, Transformer
```

Comment:
we may need to change this to 
```suggestion
from delphi.train.llama2c.model import ModelArgs, Transformer
```